lib/index.ts

```diff
@@ -91,26 +91,33 @@ const app = function (request: http.IncomingMessage, response: http.ServerRespon
     }
 }
 
-// Get the port - set to 9200 by default
-const port = process.env.SERVER_PORT ? parseInt(process.env.SERVER_PORT, 10) : 9200;
+// Get the HTTP and HTTPS port - set to 9200 by default
+const httpsPort = process.env.HTTPS_PORT ? parseInt(process.env.HTTPS_PORT, 10) : undefined;
```

Comment:
I would not rely on port to activate SSL, but in a env variable for the purpose, this might create two servers.

Reply:
That is the intention - we need two servers, one for HTTP and one for HTTPS

Reply:
Got it, then LGTM